tests/FakeItEasy.Specs/XBehaveExtensions.cs

```diff
@@ -0,0 +1,70 @@
+namespace FakeItEasy.Specs
+{
+    using System;
+    using System.Diagnostics.CodeAnalysis;
+    using Creation;
+    using Xbehave;
+    using Xbehave.Sdk;
+
+    public static class XBehaveExtensions

```

Comment:
It makes sense in the context of our specs, where we want these extension
methods on strings.
On 5 Jun 2016 10:28 p.m., "Thomas Levesque" <notifications@github.com>
wrote:

> In tests/FakeItEasy.Specs/XBehaveExtensions.cs
> <https://github.com/FakeItEasy/FakeItEasy/pull/748#discussion_r65825443>:
>
> > @@ -0,0 +1,70 @@
> > +namespace FakeItEasy.Specs
> > +{
> > +    using System;
> > +    using System.Diagnostics.CodeAnalysis;
> > +    using Creation;
> > +    using Xbehave;
> > +    using Xbehave.Sdk;
> > +
> > +    public static class XBehaveExtensions
>
> Normally I would agree with @adamralph <https://github.com/adamralph>,
> but in this case I think we can make an exception to the rule, because this
> method only makes sense in the context of xBehave
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/FakeItEasy/FakeItEasy/pull/748/files/88be5d1edac67e6718c16abed289528b6562e50c#r65825443>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AApXSDlxSiiLD3cL3QLfu-32eNrEu_1Cks5qIz9sgaJpZM4IuVk0>
> .
>